src/OrchardCore.Modules/OrchardCore.Email/Views/Items/EmailTask.Fields.Edit.cshtml

```diff
@@ -73,12 +82,36 @@
 <script asp-src="~/OrchardCore.Liquid/codemirror/liquid.js" at="Foot"></script>
 
 <script at="Foot">
-$(function () {
-    var bodyEditor = CodeMirror.fromTextArea(document.getElementById('@Html.IdFor(x => x.Body)'), {
-        lineNumbers: true,
-        styleActiveLine: true,
-        matchBrackets: true,
-        mode: { name: "liquid" },
+    $(function () {
+        CodeMirror.fromTextArea(document.getElementById('@Html.IdFor(m => m.HtmlBody)'), {
+            lineNumbers: true,
+            styleActiveLine: true,
+            matchBrackets: true,
+            mode: { name: "liquid" },
+        });
+
```

Comment:
```suggestion
        CodeMirror.fromTextArea(document.getElementById('@Html.IdFor(m => m.TextBody)'), {
            lineNumbers: true,
            styleActiveLine: true,
            matchBrackets: true,
            mode: { name: "liquid" },
        });
```
Now there is no editor for the plain text body anymore.

Reply:
Why it's needed while it's simple plain text?!!

Reply:
Because it supports Liquid.

Reply:
Ya, I notice that CodeMirror supports liquid, you're right